db/schema.sql

```diff
@@ -30,7 +30,8 @@ create table users (
     is_superuser    boolean not null default 'f',
     title           text,
     twitter_id      bigint  unique,
-    facebook_id     bigint  unique
+    facebook_id     bigint  unique,
+    area_id         integer
```

Comment:
Whilst we don't need to query by area right now, mysociety/fixmystreetforcouncils#38 makes it a likely requirement in the near future so I think this is best as its own column.